### ISSUE TITLE
Registry identifier is not registered when using the registration feature

### ISSUE THREAD

**Issue by karlsaito** [SaitoTech/saito-lite-rust#164]
Duplicate of #215 

**Test Steps**:
1. Open Arcade
2. Click Anonymous Account
3. Registry Modal will appear
4. Type email name in the input section
5. Click Register username
6. A notification will appear
7. anonymous account is replaced with Saito Email address

Issue:
1. Anonymous account is not updated with Saito email address

**Comment by karlsaito** [comment]
Second node is rejecting the transaction

Dependent on #155

**Comment by trevelyan** [comment]
This should be fixed. But it needs to be tested live. Will do that today.

The issue is caused because lite-clients do not receive all transactions in a block, thus do not receive notifications that other users have registered a username. We are now having servers proactively notify connected lite-clients on receipt of these notifications, and then having them register the usernames in real-time.

There is still the delay on username registration, but if you register a username it will show up for other users who are already live on the site.
